cluster-http-tests/src/test/scala/akka/cluster/http/management/MultiDcSpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster.http.management
+
+import akka.actor.ActorSystem
+import akka.cluster.Cluster
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{ HttpRequest, StatusCodes }
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.ActorMaterializer
+import akka.testkit.SocketUtil
+import com.typesafe.config.ConfigFactory
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.time.{ Millis, Seconds, Span }
+import org.scalatest.{ Matchers, WordSpec }
+
+class MultiDcSpec extends WordSpec with Matchers with ScalaFutures with ClusterHttpManagementJsonProtocol {
+
+  implicit val patience = PatienceConfig(timeout = Span(5, Seconds), interval = Span(50, Millis))
+
+  val config = ConfigFactory.parseString(
+    """
+      |akka.actor.provider = "cluster"
+      |akka.remote.log-remote-lifecycle-events = off
+      |akka.remote.netty.tcp.hostname = "127.0.0.1"
+      |#akka.loglevel = DEBUG
+    """.stripMargin
+  )
+
+  "Http cluster management" must {
+    "allow multiple DCs" in {
+      val httpPortA = SocketUtil.temporaryServerAddress().getPort
+      val portA = SocketUtil.temporaryServerAddress().getPort
+      val portB = SocketUtil.temporaryServerAddress().getPort
+      val dcA = ConfigFactory.parseString(
+        s"""
+           |akka.cluster.http.management.hostname = "127.0.0.1"
+           |akka.cluster.http.management.port = $httpPortA
+           |akka.cluster.seed-nodes = ["akka.tcp://MultiDcSystem@127.0.0.1:$portA"]
+           |akka.cluster.multi-data-center.self-data-center = "DC-A"
+           |akka.remote.netty.tcp.port = $portA
+          """.stripMargin
+      )
+      val dcB = ConfigFactory.parseString(
+        s"""
+           |akka.cluster.seed-nodes = ["akka.tcp://MultiDcSystem@127.0.0.1:$portA"]
+           |akka.cluster.multi-data-center.self-data-center = "DC-B"
+           |akka.remote.netty.tcp.port = $portB
+          """.stripMargin
+      )
+
+      implicit val dcASystem = ActorSystem("MultiDcSystem", config.withFallback(dcA))
+      val dcBSystem = ActorSystem("MultiDcSystem", config.withFallback(dcB))
+      implicit val materialiser = ActorMaterializer()
+
+      try {
+        val cluster = Cluster(dcASystem)
+        ClusterHttpManagement(cluster).start().futureValue
+        val response = Http().singleRequest(HttpRequest(uri = s"http://127.0.0.1:$httpPortA/members")).futureValue
+        response.status should equal(StatusCodes.OK)
+        val members = Unmarshal(response.entity).to[ClusterMembers].futureValue
+        members.members.size should equal(2)
```

Comment:
I could test more, before this it blew up entirely

Reply:
so it was already fixed in master?

Reply:
Yep, by accident by the looks of it: https://github.com/akka/akka-management/commit/e4fc9d5849342590daba3495597f68bd68887f9c

Reply:
If that's the fix are we sure it's sufficient? It looks like it'd still blow up when members from different dc's are in the status `MemberStatus.Up` (https://github.com/akka/akka-management/commit/e4fc9d5849342590daba3495597f68bd68887f9c#diff-c6868e05e6924399e82f52ce16d1a10cR73)

Reply:
Ah sorry already fixed further in https://github.com/akka/akka-management/pull/52/files#diff-c6868e05e6924399e82f52ce16d1a10cL74